core/src/window.rs

```diff
@@ -146,6 +146,39 @@ impl Window {
       .focus_next_widget(&self.widget_tree.borrow().arena);
   }
 
+  /// Insert a overlay widget to the window and return the control of the
```

Comment:
I'd like to use an overlay like this:

```rust


fn_widget! {
  let menu = overlay! {
    @List {
      ...
    }
  };

  @Button {
    on_tap: move |_| overlay.show(ctx!()),
    ...
  }
}
```

We can use the `overlay` macro to create an overlay widget. This macro only requires the `ctx` (context) when the overlay is displayed.